reana/cli.py

```diff
@@ -1984,18 +2129,25 @@ def run_example(
     "mounts",
     multiple=True,
     callback=volume_mounts_to_list,
-    help="Which local directories would be mounted in the Kind nodes? local_path:cluster_node_path",
+    help="Which local directories to mount in the cluster nodes? [local_path:cluster_node_path]",
 )
 @click.option(
     "--debug",
     is_flag=True,
     default=False,
     help="Should we mount the REANA source code for live code updates?",
 )
-@click.option("--worker-nodes", default=0, help="How many worker nodes? default 0")
+@click.option("--worker-nodes", default=0, help="How many worker nodes? [default=0]")
```

Comment:
I am wondering whether it would make sense to check whether `--mount` was passed whenever the user selects 2 or more nodes, otherwise REANA won't work. What do you think?

Reply:
OK, I can add it.  (And also delete the dangling image commit...)